src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -799,16 +799,77 @@
     <Field Name="SwitchSections" Type="ImmutableArray&lt;BoundPatternSwitchSection&gt;"/>
     <Field Name="DefaultLabel" Type="BoundPatternSwitchLabel" Null="allow"/>
     <Field Name="BreakLabel" Type="GeneratedLabelSymbol"/>
-    
-    <!-- We store the pattern switch binder so we can perform some binding-like
-         operations during control flow analysis. -->
-    <Field Name="Binder" Type="PatternSwitchBinder"/>
+
+    <!-- A directed acyclic decision graph for the switch statement. It can be traversed
+    to determine if all labels are reachable, and if every input is handled. -->
+    <Field Name="DecisionDag" Type="BoundDecisionDag" Null="allow"/>
```

Comment:
>DecisionDag [](start = 17, length = 11)

This will be removed in a future PR, and a new node will be added for recursive pattern switches. Eventually this node will be removed.